test/param/shortCircuitParamRight.chpl

```diff
@@ -0,0 +1,14 @@
+config param flag = false;
```

Comment:
Worth a nod to #20496 and cross-reference with test/statements/conditionals/param/nonparam-and-false.chpl